deploy-sdn/calico/calico-2.6.1-v1.7.5.yaml

```diff
@@ -1,3 +1,121 @@
+# Calico Version v2.6.1
+# https://docs.projectcalico.org/v2.6/releases#v2.6.1
+# This manifest includes the following component versions:
+#   calico/node:v2.6.1
+#   calico/cni:v1.11.0
+#   calico/kube-controllers:v1.0.0
+
+# This ConfigMap is used to configure a self-hosted Calico installation.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config
+  namespace: kube-system
+data:
+  # The location of your etcd cluster.  This uses the Service clusterIP
+  # defined below.
+  etcd_endpoints: "http://10.96.232.136:6666"
```

Comment:
this is the example I used in my previous comment. needs to use:
```
etcd_endpoints: "http://ETCD_SVC_ADDR:ETCD_SVC_PORT"`
```

use the var from `.bootkube_env`.